lib/language_pack/ruby.rb

```diff
@@ -115,6 +115,27 @@ def compile
 
 private
 
+  def default_malloc_arena_max?
+    return true if stack_not_14_not_16?
+    return true if "true" == @metadata.read("default_malloc_arena_max")
+
+    if new_app?
+      @metadata.write("default_malloc_arena_max", "true")
+      return true
+    end
+
+    return false
+  end
+
+  def stack_not_14_not_16?
+    case stack
+    when "cedar-14", "heroku-16"
+      return false
```

Comment:
fwiw you don't _need_ `return` inside the case since it's the last evaluated line.